docs/source/DEVELOPER.rst

```diff
@@ -144,3 +144,29 @@ The Deepcell Kiosk uses Google Kubernetes Engine to requisition resources on Goo
 5. any Persistent Disks associated with your cluster
 
 While we hope this list is comprehensive, there could be some lingering resources used by Google Cloud and not deleted automatically that we're not aware of.
+
+Benchmarking Figures from DeepCell Kiosk Paper
+---------------------------------------------------------
+
+The DeepCell Kiosk paper (Bannon et al., 2020) presents cost and runtime benchmarks for running a generic segmentation pipeline on datasets of given sizes inside the DeepCell Kiosk with different sets of constraints (Fig. 1b).
+
+Here, we provide any interested parties with a way to recreate these figures.
+
+Recreating Figures
+^^^^^^^^^^^^^^^^^^
+
+1. First, checkout the `benchmarks` branch of the `kiosk-console` repository. This branch is almost identical to the version 1.2.0 release of the `kiosk-console` repo, except for minor configuration differences, which the user can view by executing `git diff master..benchmarks` from a command prompt inside a local copy of the repository. There are three very important variable settings worth noting, each with a corresponding setup action for the user to perform before benchmarking:
+    - In the `Dockerfile`, the `CLOUDSDK_BUCKET` environmental variable is set to `kiosk-benchmarking`, which is the Google Cloud bucket that the Van Valen Lab uses to store its benchmarking models. To recreate our benchmarking data, please change this value to the name of a bucket that you control.
+    - In the benchmarking pod's YAML file, `conf/helmfile.d/0410.benchmarking.yaml`, the `MODEL` environmental variable has been set to the exact model name and version we used in our benchmarking, `NuclearSegmentation:2`. The benchmarking pod will look for the model files corresponding to this name and version in the `models/` folder of your bucket. To recreate our benchmarking data, please copy all the files form the `models/NuclearSegmentation` folder in the `kiosk-benchmarking` bucket to the `models/NuclearSegmentation` folder in your bucket.
+    - Also in the benchmarking pod's YAML file is an environmental variable, `FILE`, which is set to `zip100.zip`. The benchmarking pod will look for a file by this name in the bucket you specified above at the location `uploads/[file_name]`. We have a file named `zip100.zip` located in our `kiosk-bencharking` bucket at `sample_data/zip100.zip`. This file consists of 100 microscopy images, which we used as the basis for all our benchmarking runs. To recreate our benchmarking data, please copy `sample_data/zip100.zip` from the `kiosk-benchmarking` bucket to `uploads/zip100.zip` in your bucket.
```

Comment:
Again, the benchmarking pod itself does not look to see it is there. Instead, it just creates a job assuming that the file exists. The redis consumer is the pod that actually checks for the file and downloads it.

Reply:
I think I removed this reference.